flow/project.py

```diff
@@ -493,21 +592,43 @@ def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
                 "must be a member of class IgnoreConditions")
         # len(self._prereqs) check for speed optimization
         pre = (not len(self._prereqs)) or (ignore_conditions & IgnoreConditions.PRE) \
-            or all(cond(job) for cond in self._prereqs)
+            or all(cond(jobs) for cond in self._prereqs)
```

Comment:
I thought we had agreed that condition functions have signature `cond(*jobs)`?

Reply:
This method is internal hence we pass in a list of jobs here.
As `_FlowCondition` is also internal, @b-butler suggested me to pass in a list of jobs while calling `_FlowCondition` and then unpack the value while passing the jobs to the function.

Reply:
👍